tox.ini

```diff
@@ -4,20 +4,26 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py{27,34,35,36}, lulesh, nose
+envlist = flake, py{27,34,35,36}
 skip_missing_interpreters=True
 
 [travis]
 python =
-	2.7: py27, nose, lulesh
-	3.4: py34, nose, lulesh
-	3.5: py35, nose, lulesh
-	3.6: py36, nose, lulesh
+	2.7: py27
+	3.4: py34
+	3.5: py35
+	3.6: py36
 
-[testenv:nose]
-deps = nose
-commands =
-	nosetests
+[testenv]
+deps:
+	pytest
+commands:
+	pytest {posargs}
```

Comment:
I had initially planned to use nose for testing -- to my understanding, people prefer one or the other usually but I recall reading that nose was not being as supported or something to that effect. I'll have to google around to catch up on what is considered standard. I don't personally have a preference either way, but we can sort that out later.

Related, since we now require pytests, we need to include it in the `requirements.txt` so that developers will automatically pull it when they setup initially.

Reply:
Pytest is added to the requirements.txt.

As for the pytest/nose I prefer pytest, but do not care either way. I did run these tests through nose and they worked there as well so switching should not be a problem.

Reply:
Oh -- I was in the wrong environment. There's pytest. We'll stick with pytest for now then, I just removed nose from the requirements.